src/nvme/util.h

```diff
@@ -560,7 +560,7 @@ char *kv_keymatch(const char *kv, const char *key);
  */
 char *startswith(const char *s, const char *prefix);
 
-#define min(x, y) ((x) > (y) ? (y) : (x))
+#define nvme_min(x, y) ((x) > (y) ? (y) : (x))
```

Comment:
I think we should not expose this macro at all. And there should be already the min macro from the ccan 'library'. Can you drop this one and include the ccan/minmax.h header in the c files?

Reply:
yup, we can, but we need cast the type, see updated code.